msft_Azure_IoT/demo/msft_Azure_IoT_clientcredential.h

```diff
@@ -64,5 +64,39 @@
  */
 #define clientcredentialAZURE_MQTT_BROKER_PORT             8883
 
+/*
```

Comment:
I love this comment!

Reply:
The first half of it is your comment so thank you! I just tried to continue the instructions in a similar fashion.